backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -279,10 +272,6 @@ public RepositoryEntity getRepositoryById(
                           "Couldn't find repository by id " + id.getRepoId(), Code.NOT_FOUND));
     }
     try {
-      if (checkProtected && repository.isProtected()) {
-        throw new ModelDBException(
-            "Can't access repository because it's protected", Code.PERMISSION_DENIED);
-      }
```

Comment:
I do not think this is right, we had this because we did not want the protected repos to be accessed via RepoService

Reply:
Ok, Your point is right. I misunderstood that point.